test/get-module-specifier-test.js

```diff
@@ -59,6 +59,17 @@ describe('get-module-specifier', function() {
     );
   });
 
+  it('identifies named module with a dot delimeter', function() {
+    let moduleExtension = '.hbs';
+
+    assert.equal(getModuleSpecifier(modulePrefix, moduleConfig, 'ui/components/text-editor.glimmer', moduleExtension),
+      'template:/my-app/components/text-editor.glimmer'
```

Comment:
@robbiepitts The specifiers are already eliding a number of things from the module paths (e.g. we drop `ui` out of all things under `src/ui`).